clients/unshaded/src/main/java/tachyon/client/table/TachyonRawTablesCore.java

```diff
@@ -0,0 +1,79 @@
+/*
+ * Licensed to the University of California, Berkeley under one or more contributor license
+ * agreements. See the NOTICE file distributed with this work for additional information regarding
+ * copyright ownership. The ASF licenses this file to You under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance with the License. You may obtain a
+ * copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under the License
+ * is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express
+ * or implied. See the License for the specific language governing permissions and limitations under
+ * the License.
+ */
+
+package tachyon.client.table;
+
+import java.io.IOException;
+import java.nio.ByteBuffer;
+
+import tachyon.TachyonURI;
+import tachyon.annotation.PublicApi;
+import tachyon.exception.TachyonException;
+import tachyon.thrift.RawTableInfo;
+
+/**
+ * User facing interface for the Tachyon Raw Table client APIs. Raw tables are collections of
+ * columns stored as files. A column may consist of one or more ordered files.
+ */
+@PublicApi
+interface TachyonRawTablesCore {
+  /**
+   * Creates a new raw table.
+   *
+   * @param path the path of the table to create in Tachyon space, this must not already exist as
+   *             a file or table.
+   * @param numColumns the number of columns in the table
+   * @param metadata the metadata associated with the table, this will be stored as bytes and
+   *                 should be in a format the user can later understand
+   * @return a {@link SimpleRawTable} satisfying the input parameters
+   * @throws IOException if a non Tachyon related I/O error occurs
+   * @throws TachyonException if an internal Tachyon error occurs
+   */
+  SimpleRawTable create(TachyonURI path, int numColumns, ByteBuffer metadata)
```

Comment:
Yeah, I put a todo for that. For this PR I'm just trying to migrate the functionality over as naively as possible so we can remove the deprecated code (1270).